### PR TITLE
Hackathon Section Enhancement #198

### PR DESCRIPTION
**Pull request by archiejain28**
Hi @anmode, 
I have enhanced the Hackathon cards by adding some CSS effects as well fixed the issues which I addressed in #198. Cards are more user interactive now. 

Below are the **enhancements** :

On hover, 
 - user will see both blue layer as well as title of the hackathon at same time. (Earlier title was only visible when you hover at center of the image).
 - Added shadow effect on the cards
 - Added card lifting effect - > The card moves 4px up and shadow casts down.
 
 Below are the **fixes** :
  - Image border's are properly aligned with blue layer now.
  

https://github.com/anmode/grabtern-frontend/assets/82926210/8fdb2fa6-230b-4a60-b209-8162ed86300d

### CODE REVIEW

**Review by anmode — CHANGES_REQUESTED** [review]
Nice, It would be great. If you make hackathon component css in a file named hackathon.css

File: public/assets/css/style.css [hunk 1]
```diff
@@ -3505,6 +3505,11 @@ hr {
 .topic-area .single-topic {
   position: relative;
 }
+.topic-area .single-topic:hover{
```
Comment (anmode) [inline public/assets/css/style.css:3508]: Please change the css class name to hackathon. We really want to refactor..

**Review by anmode — COMMENTED** [review]
Change the class names!

**Comment by anmode** [comment]
Hey! @archiejain28 please refer this pR https://github.com/anmode/grabtern-frontend/pull/224.
when you do like this. I will merge your PR before this PR. and your commits will also be increased.

**Comment by archiejain28** [comment]
Thank you @anmode , I will do the required changes.

**Comment by archiejain28** [comment]
Hi @anmode I have refactored the code. 
Could you pls review them now.

**Review by anmode — APPROVED** [review]
Wow! @archiejain28 nice work. Just last change and it is good to merge.
I see you did rounder couner... should before too we can have rounder corner and that 3d? 
On hover only two things should be happen
1. Overlay
2. Text ( No animation)

Why I am suggesting? As you did many things on hovering.. Sometimes it keep hang as on hover there is lots of after css property.

**Comment by archiejain28** [comment]
Done @anmode , as discussed - 
Removed round corners from  the hover effect & added them in the image itself.

Could pls check them now.

**Comment by anmode** [comment]
I think providing you level 3 would be nice choice. Reason: Great work

**Comment by archiejain28** [comment]
Thanks much @anmode ! :)

**Pull request merged**